storage/innobase/dict/dict0dd.cc

```diff
@@ -980,8 +979,11 @@ dict_table_t *dd_table_create_on_dd_obj(const dd::Table *dd_table,
         if (c == dd_col) break;
         if (c->is_virtual() == dd_col->is_virtual()) col_pos++;
       }
+      /* FULLTEXT and HASH indexes can have UNDEF order, we should treat UNDEF
```

Comment:
anything PXB changes on areas that comes from upstream, we should use pxb macros:

either:
``` 
#ifdef XTRABACKUP
      bool is_asc = (idx_elem->order() == dd::Index_element::ORDER_ASC ||
                     idx_elem->order() == dd::Index_element::ORDER_UNDEF);
#else
bool is_asc = (idx_elem->order() == dd::Index_element::ORDER_ASC);
#endif  //XTRABACKUP
```

or

```
bool is_asc = (idx_elem->order() == dd::Index_element::ORDER_ASC IF_XB( || idx_elem->order() == dd::Index_element::ORDER_UNDEF));
```

Reply:
this entire function is PXB specific.

Reply:
so this ifdef might not be necessary

Reply:
seems like code from before the macro rule.

Reply:
right.. since we are touching this, we can put ifdef around the entire function

Reply:
ack, will put it around the function